package.py

```diff
@@ -1,6 +1,6 @@
 name = "PIK_maya_loader"
 
-version = "0.1.1"
+version = "0.1.3"
```

Comment:
A bit strange this version jump from 1 to 3.

Reply:
If you want to bundle dwpicker and Ez_Renamer, you can just imcrement from 1 to 2.